distinct_item_types_report.csv

```diff
@@ -0,0 +1,138 @@
+"book, limited circ, MaRLI",55,Book,
+serial,3,Serial,

```

Comment:
I noticed on the csv, some item have "" such as `"book, limited circ, MaRLI",55,Book,` yet some do not such as `serial,3,Serial,`. Also some do not have the first part such as `,146,Other,`. Will this cause any potential problems? I feel we should clean the csv up a bit?

Reply:
The "" is to distinguish types where the name of the type includes a comma. We could add it to the other type names if that seems easier to read, but it shouldn't be strictly necessary